lib/createTopLevelExpect.js

```diff
@@ -1627,10 +1708,10 @@ expectPrototype.freeze = function () {
 };
 
 expectPrototype.outputFormat = function (format) {
-  this._assertTopLevelExpect();
   if (typeof format === 'undefined') {
     return this._outputFormat;
   } else {
+    this._assertTopLevelExpect();
```

Comment:
IIRC I hit an issue with this myself at some point and I think this fix may want to be applied directly to master.

Reply:
Yeah, I agree: https://github.com/unexpectedjs/unexpected/commit/5349b0fa